examples/tut-6-rpc/client-py/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM       python:3.8-alpine
+
+WORKDIR    /app
+
+# Install and cache `pip` dependencies
+RUN        python3.8 -m venv venv
```

Comment:
`venv` is less about Python version (although it sandboxes that too) and more about keeping third-party packages installed via `pip` isolated from the host system's package manager-installed versions. It's generally considered good practice to sandbox these and completely bypass dependency hell.

If you look at `pip-deps.txt` you'll see that versions are also pinned there, whereas they wouldn't be on the host system unless we did something else special.

For our containerized purposes some people might say "it's Docker, so it doesn't matter"... I figure it's easy to do it the clean way so that's generally my default. Would you rather ditch using `venv`s?

Reply:
It's fine